### DISCUSSION TITLE
Add the ability to set the default view status for specific channels and sections

### DISCUSSION THREAD

**Discussion by terryupton** [craftcms/cms#8926] [Ideas]
### Description
It would be a great addition to be able to set/specify the default status when viewing channels and structures.
At the moment they all default to 'all'. If you change the setting to 'live' then it won't recall this next time you view the structure/channel. 

So adding either the ability to set the default status within a channel/structure settings.
Or somehow tracking the last status set and reverting to this for each individual channel/structure and recalling this when logging in again.

**Comment by wildmanonline** [comment]
I second this. Changing to only "Live" (which is likely a common task for many workflows) adds 2 additional clicks _each_ time the entry listing page loads for a particular section. It may seem small, but those clicks build up over time. It's likely not a hard fix and would pretty dramatically enhance the user experience for quite a few workflows I'd imagine.

Like @terryupton mentions, adding a default status to a section would solve this.
